server/src/main/java/org/elasticsearch/tasks/TaskResult.java

```diff
@@ -79,7 +79,7 @@ public TaskResult(TaskInfo task, ToXContent response) throws IOException {
         this(true, task, null, XContentHelper.toXContent(response, Requests.INDEX_CONTENT_TYPE, true));
     }
 
-    private TaskResult(boolean completed, TaskInfo task, @Nullable BytesReference error, @Nullable BytesReference result) {
+    public TaskResult(boolean completed, TaskInfo task, @Nullable BytesReference error, @Nullable BytesReference result) {
```

Comment:
Can this stay private?

Reply:
I don't know how to do it without making security manager upset.

Reply:
Oh! Because you can't making an explicit reference to the ctor, you are digging the ctor from the public reflection information. Got it! Ignore this comment.